platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,66 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, MGLLightAnchorType) {
+    MGLLightAnchorTypeMap,
+    MGLLightAnchorTypeViewport
+};
+
+MGL_EXPORT
+@interface MGLLight : NSObject
+
+/**
+ `lightAnchorType` property.
+ */
+@property (nonatomic) MGLStyleValue<NSValue *> *lightAnchorType;
+
+/**
+ Values describing animated transitions to `lightAnchorType` property.
+ */
+@property (nonatomic) MGLTransition lightAnchorTypeTransition;
+
+
+/**
+ Position.
+ */
+@property (nonatomic) MGLPosition position;
```

Comment:
[The style specification documentation](https://www.mapbox.com/mapbox-gl-js/style-spec/#light-position) for this property seems to indicate that it can be set to a function. So the type needs to be MGLStyleValue. Since MGLPosition is a struct rather than a class, we’ll need to use the type `MGLStyleValue<NSValue *> *` and provide some extensions to NSValue to convert between NSValue and MGLPosition.